kata-deploy/examples/test-deploy-kata-qemu-virtiofs.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    run: php-apache-kata-qemu-virtiofs
+  name: php-apache-kata-qemu-virtiofs
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      run: php-apache-kata-qemu-virtiofs
+  template:
+    metadata:
+      labels:
+        run: php-apache-kata-qemu-virtiofs
+    spec:
+      runtimeClassName: kata-qemu-virtiofs
+      containers:
+      - image: k8s.gcr.io/hpa-example
+        imagePullPolicy: Always
+        name: php-apache
+        ports:
+        - containerPort: 80
+          protocol: TCP
+        resources:
+          requests:
+            cpu: 200m
+      restartPolicy: Always
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: php-apache-kata-qemu-virtiofs
```

Comment:
We dont have a test today for this you better try locally to see if we are not missing something here :)